_saas-integrations/saasoptics/saasoptics-v1.md

```diff
@@ -0,0 +1,120 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: SaaSOptics
+permalink: /integrations/saas/saasoptics
+keywords: saasoptics, integration, schema, etl saasoptics, saasoptics etl, saasoptics schema
+layout: singer
+# input: false
+
+key: "saasoptics-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "saasoptics"
+display_name: "SaaSOptics"
+
+singer: true
+
+tap-name: "SaaSOptics"
+repo-url: https://github.com/singer-io/tap-saasoptics
+
+this-version: "1"
+
+api: |
+  [{{ integration.display_name }} REST API](https://saasoptics.zendesk.com/hc/en-us/articles/115003595913-URLs-and-REST-Info){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: false 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+api-type: "platform.saasoptics"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
```

Comment:
Found a setup requirement in [these SaaSOptics docs](https://saasoptics.zendesk.com/hc/en-us/articles/115013917848-Enablement-Authentication) about API tokens:

> **Only an Administrator can actually generate the first Token for a User, but a User can regenerate the Token if needed.** For an existing User, navigate to the User in SaaSOptics (Admin>User) and select the API menu option when viewing the User. Check the box marked "(Re)Generate Token."

I'll explain it if it's unclear, but this is a good research/learning opportunity. What does this mean?

Two changes are needed here - let's chat about them if you're not sure:

1. There should be a requirements list that outlines this - what would that look like?
2. This will also change the instructions for Step 2 - any idea how?